libs/application/core/src/lib/fieldBuilders.ts

```diff
@@ -119,6 +119,8 @@ export function buildRadioField(data: {
   description?: MessageDescriptor | string
   options: Option[]
   required?: boolean
+  emphasize?: boolean
+  largeButtons?: boolean
```

Comment:
I am just wondering, would you ever want to emphasize without having large buttons? 

what I am trying to get at, is maybe this should just be one attribute? 

I do not have a strong preference, it is fine as is

Reply:
I thought about this as well :) There will be a case where you want just large buttons without emphasis:
![image](https://user-images.githubusercontent.com/8889977/95479157-fcafbb00-0979-11eb-856a-65ece4cabd94.png)

But we could simplify, that if you have `emphasize` you won't have to set 'largeButtons'...

Reply:
nono, that is too much. this alone is enough of a case for having both props :)

Reply:
Updated it now.

Reply:
hehe, oops checked it in, it's a one liner in RadioController `<RadioButton large={largeButtons || emphasize}`, so will leave for now :)